snap-guest

```diff
@@ -219,13 +219,18 @@ if [ -f "$TARGET_IMG" ]; then
   if [ $FORCE -eq 1 ]; then
     REPLY="y"
   else
-    read -p "Shutdown and remove host with this name first (y/[n])? "
+    read -p "Shutdown and remove host with this name first (y/[n])? " REPLY
```

Comment:
```If  no  names  are supplied, the line read is assigned to the variable REPLY.```

REPLY is not needed

Reply:
Hmm, did not knew that, thank you! Anyway, "Explicit is better than implicit" ;-)